_config/config.yml

```diff
@@ -7,4 +7,8 @@ LeftAndMain:
     - 'betternavigator/javascript/BN.LeftAndMain.Preview.js'
 ContentController:
   extensions:
-    - BetterNavigator
+    - BetterNavigatorExtension
+
+Injector:
```

Comment:
I think let's remove this part, probably no one is using BetterNavigatorExtension on anything besides ContentController  and I need to bump the major version number due to another contribution anyway, so let's include this at the same time and not worry about backwards compatibility here.